pkg/gui/style/decoration.go

```diff
@@ -0,0 +1,55 @@
+package style
+
+import "github.com/gookit/color"
+
+type Decoration struct {
+	bold      bool
+	underline bool
+	reverse   bool
+}
+
+func (d Decoration) SetBold() {
```

Comment:
I've made it so that you can only add decorations, you can't remove them. I can't think of any use cases for removing decorations off the top of my head, and if we wanted to support that we'd need to change how users configure custom colors to e.g. say 'bold: false'. But I think until we have that use case we should keep things simple by having everything be additive